indexer-app/src/cmr/indexer/services/index_service.clj

```diff
@@ -761,7 +761,23 @@
        context
        index
        (concept-mapping-types :service)
-       {:term {(query-field->elastic-field :provider-id :service) provider-id}}))))
+       {:term {(query-field->elastic-field :provider-id :service) provider-id}}))
+
+    ;; delete the tools
```

Comment:
nit: The calls for `:variable`, `:service`, `:tool` and `:subscription` concept types can be put into a `doseq` of itself. This way, we don't have to repeat the code for doing the `delete-by-query` calls four times.

Reply:
I'm not on the review, but I agree (which means I can actually read the code enough to have an opinion).

Reply:
Done.